.github/workflows/frontend-ci.yml

```diff
@@ -158,6 +158,7 @@ jobs:
             -Dsonar.projectKey=geh-frontend-app
             -Dsonar.organization=energinet-datahub
             -Dsonar.sources=./apps/dh/app-dh,./libs/dh,./libs/ui-watt,./libs/gf
+            -Dsonar.exclusions=**/+storybook
```

Comment:
Question: Does this mean every sonar related scanning is disabled for our storybook related files. Shouldn't it only be the coverage we wanted to exclude?

Reply:
Good point. There was a setting to exclude only coverage. Will give it a try.